var/spack/repos/builtin/packages/avizo/package.py

```diff
@@ -26,12 +25,13 @@ def url_for_version(self, version):
         return "file://{0}/avizo_{1}_linux64_gcc44.tar.gz".format(os.getcwd(), version)
 
     def install(self, spec, prefix):
-        ver = str(self.version).replace(".", "")
-        os.system('sh Avizo-{0}-Linux64-gcc44.bin --noexec --keep'.format(ver))
-
-        os.chdir(join_path(self.stage.source_path, 'Avizo'))
-        avizo_tar = tarfile.open(name='Avizo-{0}-Linux64-gcc44.tar.bz2'.format(self.version))
-        avizo_tar.extractall()
-
-        install_tree(join_path(self.stage.source_path, 'Avizo/Avizo-{0}'.format(self.version)), prefix)
-        intsall('file://{0}/password.dat'.format(os.getcwd()), join_path(prefix, 'share/license')
+        ver = self.version.joined
+        sh = which('sh')
+        sh('Avizo-{0}-Linux64-gcc44.bin'.format(ver), '--noexec', '--keep')
+
+        with working_dir('Avizo'):
+            avizo_tar = tarfile.open(name='Avizo-{0}-Linux64-gcc44.tar.bz2'.format(self.version))
+            avizo_tar.extractall()
+            install_tree(join_path(self.stage.source_path, 'Avizo/Avizo-{0}'.format(self.version)), prefix)
+            intsall('password.dat', prefix.share.license)
```

Comment:
@adamjstewart I have to ask, how how spack get the correct path for the password.dat here?  I mean, it's not included some how in the tarball, we get it seperataly from vendor.

i tried to use this before as in intel recipe, but it didn't work
```
    @property
    def license_files(self):
        return [
            'Licenses/password.dat',
            join_path('share', 'license', 'password.dat')
        ]```